public/test/test-utils.tsx

```diff
@@ -17,11 +16,11 @@ interface ExtendedRenderOptions extends RenderOptions {
    * Optional store to use for rendering. If not provided, a fresh store will be generated
    * via `configureStore` method
    */
-  store?: ToolkitStore;
+  store?: Store<StoreState>;
   /**
    * Partial state to use for preloading store when rendering tests
    */
-  preloadedState?: PreloadedState<StoreState>;
+  preloadedState?: Partial<StoreState>;
```

Comment:
This means that if you want to define a piece of preloaded state for a test, you need to provide an entire matching state after the top level item 🤔 rather than just part of it

I guess this is better, but it's what I was trying to avoid previously - however, on reflection/reading around, I think it's better to have it this way, as the cases where you need to provide a preloaded state should be rarities overall

With this change, you can remove the `as any` and associated `eslint-disable` further down in this file 👇